src/org/traccar/notification/NotificationFormatter.java

```diff
@@ -88,16 +88,16 @@ public static Template getTemplate(Event event, String path) {
         return template;
     }
 
-    public static MailMessage formatMailMessage(long userId, Event event, Position position) {
-        String templatePath = Context.getConfig().getString("mail.templatesPath", "mail");
+    public static FullMessage formatFullMessage(long userId, Event event, Position position) {
+        String templatePath = Context.getConfig().getString("message.full.templatesPath", "full");
         VelocityContext velocityContext = prepareContext(userId, event, position);
         String formattedMessage = formatMessage(velocityContext, userId, event, position, templatePath);
 
-        return new MailMessage((String) velocityContext.get("subject"), formattedMessage);
+        return new FullMessage((String) velocityContext.get("subject"), formattedMessage);
     }
 
-    public static String formatSmsMessage(long userId, Event event, Position position) {
-        String templatePath = Context.getConfig().getString("sms.templatesPath", "sms");
+    public static String formatShortMessage(long userId, Event event, Position position) {
+        String templatePath = Context.getConfig().getString("message.short.templatesPath", "short");
```

Comment:
Not really related to this pull request, but I think we should just assume "full" and "short" sub-paths here. Base path can still be configurable.